ebuild-writing/variables/text.xml

```diff
@@ -435,15 +435,28 @@ that can afterwards be referenced through a <c>mirror://</c> URL:
 SRC_URI="mirror://example/${PN}/${P}.tar.gz"
 </codesample>
 
-<warning>
-If the sources for a package are neither mirror-restricted nor
-fetch-restricted, then they will be <uri
-link="::general-concepts/mirrors">mirrored onto Gentoo infrastructure</uri>
-automatically; in that case, there is little benefit to using this feature.
-There is a maintenance burden associated with using a third-party mirror list:
-it must be updated when new mirrors are born and when old mirrors die. Please
-consider that when deciding whether or not to use this feature.
-</warning>
+<p>
+There are two valid cases for using <c>thirdpartymirrors</c>:
```

Comment:
I tried to make a change like this before and it got rejected because it goes against what's currently done in ::gentoo. Needless to say, I prefer the stronger language.